crates/humanode-runtime/src/tests/fixed_supply.rs

```diff
@@ -1,13 +1,12 @@
 //! Tests to verify fixed supply logic.
```

Comment:
To ensure fixed supply we now have to check that **total bridges balance** equals **total issuance** + 2 * **existential balance** at each test.

Reply:
Added #695